gf-webapp/src/Index.scss

```diff
@@ -66,4 +66,53 @@ $input-padding: 10px;
 $input-font-size: 18px;
 $input-font-family: Montserrat Regular;
 $input-border: 2px solid $color-primary;
-$input-border-focus: 2px solid $color-secondary;
\ No newline at end of file
+$input-border-focus: 2px solid $color-secondary;
+
+//room titles styles
+$room-title-font-family: Montserrat Regular;
+
+//room h1 titles styles
+$room-h1-display: flex;
+$room-h1-flex-direction: column;
+$room-h1-align-items: center;
+$room-h1-justify-content: center;
+$room-h1-position: absolute;
+$room-h1-top: 0;
+$room-h1-left: 0;
+$room-h1-right: 0;
+$room-h1-bottom: 0;
+$room-h1-color: $color-white;
+$room-h1-font-size: 60px;
+
+@mixin room-h1 {
```

Comment:
change naming, all h1 should be the same. For example, we couldn't have different styles for room-h1 and contacts-h1

Reply:
But we have)
The only same properties between room-h1 and contacts-h1 are font-size: 60px and font-family: $title-font-family;
![Contacts](https://user-images.githubusercontent.com/103531486/236170901-24532829-c511-4ab2-bdc2-ea5e9940479e.png)
![Rooms](https://user-images.githubusercontent.com/103531486/236170906-bf20e930-454f-492d-abdc-f94596b8d2b3.png)
I mix them like this
```
$title-font-family: Montserrat Regular;

//h1 titles styles
@mixin h1 {
  font-size: 60px;
  font-family: $title-font-family;
}

@mixin h1-promo {
  @include h1;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  position: absolute;
  inset: 0 0 0 0;
}
```